services/dependabot/dependabot.service.js

```diff
@@ -58,7 +58,6 @@ module.exports = class DependabotSemverCompatibility extends BaseJsonService {
     return {
       color: json.colour,
       message: json.status,
-      link: this._getLink({ packageManager, dependencyName }),
```

Comment:
Do we still need to keep the `_getLink()` function itself? Seems like we could remove it..

Reply:
Indeed, had missed it, good catch.